commit f4a20381874486f6cc0448f75da923b8726a2936
Author: Steve Wheeler <stevew@netgate.com>
Date:   Fri Aug 22 17:27:08 2025 +0000

    Correctly mount multiple SWAP devices. Fixes #16232
    
    Correct the detection of SWAPDEVICE when the fstab contains multiple lines including SWAP.

diff --git a/src/etc/pfSense-rc b/src/etc/pfSense-rc
index 6d975ac23c..48f18023d4 100755
--- a/src/etc/pfSense-rc
+++ b/src/etc/pfSense-rc
@@ -393,7 +393,7 @@ echo
 
 /sbin/conscontrol mute off >/dev/null
 
-SWAPDEVICE=`/bin/cat /etc/fstab | /usr/bin/grep swap | /usr/bin/awk '{ printf $1 }' | /usr/bin/head -n 1`
+SWAPDEVICE=`/bin/cat /etc/fstab | /usr/bin/grep swap | /usr/bin/awk '{ print $1 }' | /usr/bin/head -n 1`
 if [ -n "${SWAPDEVICE}" ]; then
 	/bin/rm -f /tmp/fstab.swap
 	if ! [ -c "${SWAPDEVICE}" ]; then
